packages/snowpack/src/index.ts

```diff
@@ -83,10 +84,12 @@ export async function cli(args: string[]) {
     config: loadAndValidateConfig(cliFlags, pkgManifest),
     lockfile: await readLockfile(cwd),
     pkgManifest,
-    logLevel: cliFlags.logLevel || 'info',
+    logger,
   };
 
   logger.level = cliFlags.logLevel || 'info';
+  if (cliFlags.debug) logger.level = 'debug';
+  if (cliFlags.silent) logger.level = 'silent';
```

Comment:
Notice how we’re still using the exact same API as pino to set the level! 🎉 

Also we added `--debug` and `--silent` flags as aliases.

Reply:
Love it.

bike-shedding: slight personal preference for "--verbose" and "--quiet". fwiw I also think these are more popular/common

more strongly held preference: remove `logLevel` as a configurable option and only support the alias flags. There are so many possible options here, I wouldn't feel confident that we could support and test each well.  At the very least, I'd want to remove it to start and then add it back in if someone asks for it (YAGNI, basically).

Reply:
I’m OK with that if we remove `--log-level`. The main thing I wanted to prevent was using different terms for the same thing, e.g. `--log-level=silent` == `--quiet`.